README.md

```diff
@@ -16,17 +16,63 @@
+
+Nussknacker can be used with its own, lightweight, Kubernetes-based [engine](https://nussknacker.io/documentation/about/engines/). It provides scalability and high availability while being straightforward to set up and maintain. However, in advanced streaming use cases, when aggregations in time windows are needed, Nussknacker can execute decision scenarios on Flink – one of the most advanced platforms for stream processing.
+
+## Why Nussknacker
+
+Nussknacker promises to make developing and deploying real-time decision algorithms as easy as it is with spreadsheets and data at rest. If so, domain experts and developers can focus on tasks that each of these two groups is most happy to perform - domain experts can author the decision algorithms and developers can solve problems beyond the reach of tools like Nussknacker.
+
+We discovered that several factors heavily influence the development of algorithms that work with real-time data:
+- Often, these are domain experts who conceptualize the algorithms and the expertise required is very domain-specific. Without proper tools for converting algorithms to code, domain experts have to delegate this work to programmers who are proficient in multiple tools, programming languages, and technologies. This approach costs money and takes time. With Nussknacker domain experts can build the algorithm from prefabricated blocks. The trick is to make these prefabricated blocks infinitely flexible -  Nussknacker achieves this by using [SpEL](https://nussknacker.io/documentation/docs/scenarios_authoring/Intro/#spel), an easy-to-learn expression language, to express data transformations and control the flow of the algorithm.
+- The algorithms may require a lot of experimentation before one gets them right. If so, the iteration time required to implement a change, deploy it, and see the result should be in single minutes if not seconds. With Nussknacker, unless the change is significant, non-technical users can achieve iteration time below one minute.
+- To support high productivity, low-code tools must provide features developers use in professional IDEs - Nussknacker Designer has built-in [syntax checking, code completion](https://nussknacker.io/documentation/about/KeyFeatures/#smart-code-suggestions-and-validation), [debugging, and testing support](https://nussknacker.io/documentation/docs/next/scenarios_authoring/TestingAndDebugging/).
+- Experimenting with algorithms requires observability going beyond pure technical metrics like throughput, Kafka topics lag, etc. Out of the box, Nussknacker comes with an integrated and ready-to-use monitoring subsystem which allows to monitor not only technical aspects of the working algorithm but also its internal behavior - for example [events count](https://nussknacker.io/documentation/docs/next/scenarios_authoring/TestingAndDebugging/#watching-nodes-filtering-behaviour-with-counts) per algorithm node. You will not need to spend developers' time on this functionality.
```

Comment:
`algorithm node` - I don't think anyone will get what's this. I haven't ever seen the usage word "node" to describe a step (or instruction) in the algorithm.

Reply:
Changed "algorithm node" to "algorithm step".